p8_integration_tests/quick_test/test_auto_pause_and_resume_tests/test_long_run/spynnaker.cfg

```diff
@@ -5,22 +5,6 @@ max_sdram_allowed_per_chip = 1529836
 [Buffers]
 use_auto_pause_and_resume = True
 # Lower than defaults
-# This cause less partitioning and therefor more auto pause runs
+# This cause less partitioning and therefore more auto pause runs
```

Comment:
"This causes", and probably "auto-pause and resume" instead of just "auto pause".

Reply:
Fixed